node-src/tasks/initialize.ts

```diff
@@ -62,6 +64,7 @@ export const announceBuild = async (ctx: Context) => {
         patchBaseRef,
         patchHeadRef,
         preserveMissingSpecs,
+        gitUserEmailHash: emailHash(gitUserEmail),
```

Comment:
```suggestion
        ...(gitUserEmail && { gitUserEmailHash: emailHash(gitUserEmail) }),
```
Otherwise we'll potentially get a hash of an empty string.